goms-application/src/main/kotlin/com/goms/v2/domain/account/usecase/QueryAccountProfileUseCase.kt

```diff
@@ -0,0 +1,42 @@
+package com.goms.v2.domain.account.usecase
+
+import com.goms.v2.common.annotation.UseCaseWithTransaction
+import com.goms.v2.common.util.AccountUtil
+import com.goms.v2.domain.account.data.dto.ProfileDto
+import com.goms.v2.domain.account.data.dto.StudentNumberDto
+import com.goms.v2.domain.auth.exception.AccountNotFoundException
+import com.goms.v2.repository.account.AccountRepository
+import com.goms.v2.repository.late.LateRepository
+import com.goms.v2.repository.outing.OutingBlackListRepository
+import com.goms.v2.repository.outing.OutingRepository
+
+@UseCaseWithTransaction
+class QueryAccountProfileUseCase(
+    private val accountUtil: AccountUtil,
+    private val lateRepository: LateRepository,
+    private val outingRepository: OutingRepository,
+    private val accountRepository: AccountRepository,
+    private val outingBlackListRepository: OutingBlackListRepository
+) {
+
+    fun execute(): ProfileDto {
+        val accountIdx = accountUtil.getCurrentAccountIdx()
+        val lateCount = lateRepository.countByAccountIdx(accountIdx)
```

Comment:
DB에 존재하지 않은 회원이라면 lateCount 변수 값을 DB에서 할당 할 필요가 없는거 같은데 해당 줄을 
`val account = accountRepository.findByIdOrNull(accountIdx) ?: throw AccountNotFoundException()` 여기 아래로 하면 어떨까요?
```suggestion
        val account = accountRepository.findByIdOrNull(accountIdx) ?: throw AccountNotFoundException()
        val lateCount = lateRepository.countByAccountIdx(accountIdx)
```

Reply:
b944abb1ac11148948ddec3d513be2f572c46489 수정했습니다.